From 2ad8cf71905217ac6a00e55598ab552f56e6b45e Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 28 Apr 2020 14:44:50 -0700
Subject: [PATCH] Set prefix to /usr

---
 qsstv/qsstv.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qsstv/qsstv.pro b/qsstv/qsstv.pro
index 56b1cab..5ffc569 100644
--- a/qsstv/qsstv.pro
+++ b/qsstv/qsstv.pro
@@ -545,7 +545,7 @@ QMAKE_EXTRA_TARGETS   +=   dox
 
 
 isEmpty(PREFIX) {
-        PREFIX = /usr/local
+        PREFIX = /usr
     }
 target.path = $$PREFIX/bin
 
-- 
2.26.2

